apis/v1alpha2/util/translator/httproute.go

```diff
@@ -14,20 +14,20 @@ See the License for the specific language governing permissions and
 limitations under the License.
 */
 
-package utils
+package translator
 
 import (
 	gatewayv1a2 "sigs.k8s.io/gateway-api/apis/v1alpha2"
 )
 
-// PathMatchTypePtr translates a string to *PathMatchType
-func PathMatchTypePtr(s string) *gatewayv1a2.PathMatchType {
-	result := gatewayv1a2.PathMatchType(s)
+// HeaderMatchTypePtr translates a string to *PathMatchType
```

Comment:
these are really trivial - calling the helper takes almost as many characters as just doing the case and obscures what is happening.  If I saw someone calling this, I would feel obligated to go look at the impl.

Is this really justified?

Reply:
i think it should be replaced by just `gatewayv1a2.HeaderMatchType(s)` and we have a helper pointer->string func somewhere in utils.

Reply:
Removed these functions in f8ca3ac3bd98413b4e71a679ed36561de51064f5 of #1569